tests/engines/registries/acceptance/draft/draft-test.ts

```diff
@@ -106,6 +107,43 @@ module('Registries | Acceptance | draft form', hooks => {
         assert.ok(currentURL().includes(`/registries/drafts/${registration.id}/2-`), 'At second (last) page');
     });
 
+    test('mobile navigation works', async assert => {
+        const initiator = server.create('user', 'loggedIn');
+        const registrationSchema = server.schema.registrationSchemas.find('testSchema');
+        const registration = server.create(
+            'draft-registration', { registrationSchema, initiator },
+        );
+
+        await visit(`/registries/drafts/${registration.id}/`);
+        setBreakpoint('mobile');
+
+        assert.ok(currentURL().includes(`/registries/drafts/${registration.id}/1-`), 'At first schema page');
+
+        // Check header
+        assert.dom('[data-test-page-label]').containsText('First page of Test Schema');
+
+        // Check next page arrow
+        assert.dom('[data-analytics-name="Sidenav back"]').isNotVisible();
+        assert.dom('[data-analytics-name="Sidenav next"]').isVisible();
+        await click('[data-analytics-name="Sidenav next"]');
```

Comment:
are there reasons why you chose to use data-analytics-* as opposed to `data-test-*` selectors?